packages/x-telemetry/package.json

```diff
@@ -13,6 +13,9 @@
     "access": "public",
     "directory": "build"
   },
+  "exports": {
+    "./package.json": "./package.json"
```

Comment:
This will be added automatically in the latest monorepo, but requires a lot more changes to update